transposon/genotypeClones.py

```diff
@@ -179,6 +179,54 @@ def pruneEdgesLowPropOfReads(G, minPropOfBCReads, minPropOfCellReads):
     pruneOrphanNodes(G)
 
 
+def computeKeyRate(G, nodes, key):
```

Comment:
can you add a 1-line description of this function, especially it's return value?

Reply:
Added "Computes and return UMI proportion of a [key] attribute value among a set of [nodes]"

Reply:
but what's the structure of the return variable?

Reply:
How about "Computes and return a dictionary of the [key] attribute value UMI proportion among a set of [nodes]"?

Reply:
How about:
Within a set of nodes, return a dictionary mapping key attribute values to the proportion of total UMIs each attribute value represents.

Reply:
Sure